404.html

```diff
@@ -13,27 +12,46 @@
     title: Careers and Employment
     description: Meaningful employment and career development for Veterans and their families.
 ---
-
-  <div class="splash">
-    <div class="row">
-      <div class="small-12 columns">
-        <div class="pitch">
-          <h2 class="tagline"><span>The page you are looking for could not be found.</span></h2>
+{% include header.html search='false' %}
+<div id="content" class="interior">
+  <div class="main home" role="main">
+    <div class="section three">
+      <div class="primary">
+        <div class="row">
+          <div class="small-12 medium-10 medium-offset-1 columns text-center usa-content">
+            <h3>Sorry — we can't find that page</h3>
+            <p>
+              Try the search box or one of the common questions below.
+            </p>
+            <div class="call-out">
+              <form accept-charset="UTF-8" action="https://search.vets.gov/search" id="search_form" class="full-width" method="get">
+                <div style="margin:0;padding:0;display:inline">
+                  <input name="utf8" type="hidden" value="&#x2713;" /></div>
+                <input id="affiliate" name="affiliate" type="hidden" value="vets.gov_search" />
+                <label for="mobile-query">Search:</label>
+                <div class="row collapse">
+                  <div class="small-9 columns">
+                    <input autocomplete="off" class="usagov-search-autocomplete full-width" id="mobile-query" name="query" type="text" />
+                  </div>
+                  <div class="small-3 columns">
+                    <input name="commit" type="submit" value="Search" />
+                  </div>
+                </div>
+              </form>
+            </div>
+          </div>
         </div>
       </div>
     </div>
-  </div>
-
-<div class="main home" role="main">
-
-  <div class="section main-menu">
-    <div class="row">
-      <div class="small-12 columns">
-        <h3>Please try one of the options listed below.</h3>
-        {% include main-navigation.html %}
-      </div>
+    <div class="section two">
+      {% include common-questions.html %}
```

Comment:
The only concern that I have here is if we should keep this as `{% include common-and-popular.html %}` so we have consistency throughout the rest of the site. Would love thoughts from @goldenmeanie and @melwoodard on this.